package.json

```diff
@@ -4,15 +4,13 @@
   "private": true,
   "dependencies": {
     "prop-types": "^15.6.0",
-    "react": "^16.0.0",
+    "react": "^16.1.0",
     "react-dom": "^16.0.0",
     "react-redux": "^5.0.6",
     "react-router-dom": "^4.2.2",
-    "react-scripts": "1.0.14",
+    "react-scripts": "1.0.17",
     "redux": "^3.7.2",
-    "styled-components": "^2.2.1",
-    "yarm": "^0.4.0",
```

Comment:
I didn't found if this dependency is being used. 🙏 please confirm

Reply:
Yep, you're right.